internal/handler/nodes_statistic_handler.go

```diff
@@ -0,0 +1,34 @@
+package handler
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/syndtr/goleveldb/leveldb"
+
+	"github.com/sorenvonsarvort/velas-sphere/internal/enum"
+)
+
+func NewGetNodesStatisticHandler(config Config) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if config.DB == nil {
+			log.Println("no db provided")
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		db := config.DB
+
+		entry, err := db.Get([]byte(enum.NodesStatisticKey), nil)
+		if err != nil {
```

Comment:
Why not to reduce nesting levels using a switch statement?

https://play.golang.org/p/2nsZw8rFFdg